slave.mk

```diff
@@ -362,6 +362,15 @@ CROSS_COMPILE_FLAGS := CGO_ENABLED=1 GOOS=linux GOARCH=$(GOARCH) CROSS_COMPILE=$
 
 endif
 
+ifeq ($(CROSS_BUILD_ENVIRON),y)
```

Comment:
I'm not sure if you understand CROSS_BUILD_ENVIRON correctly.
Do you try to build arm SONiC image on amd64 machine?

Reply:
No, I'm trying install toolchain if user want to do cross build.